scripts/build_all_fuzzers.sh

```diff
@@ -5,6 +5,9 @@ cd "$SCRIPT_DIR/.."
 
 # TODO: This should be rewritten in rust, a Makefile, or some platform-independent language
 
+# Dependencies
+sudo apt install -y libc6-dev
```

Comment:
This doesn't belong here imho

Reply:
yes should be in build_and_test.yml, and it's not needed